docs/pages/web/button.js

```diff
@@ -34,6 +32,15 @@ const PREVIEW_HEIGHT = 300;
 export default function DocsPage({ generatedDocGen }: DocType): Node {
   return (
     <Page title={generatedDocGen?.displayName}>
+      <SlimBanner
+        message={"If you're intend to use the Button for a link, see "}
```

Comment:
If you intend to use the Button as a link, see

Reply:
Changed the message, will explore ButtonLink implementation